GitHub Action fails upon authentication with SWORD API

After merging the recent pull request #201 by @shoeffner, one test fails even though all checks have passed in the pull request CI, which is quite unexpected. I am opening this issue to keep track of this.

[Failed run](https://github.com/gdcc/pyDataverse/actions/runs/10554848132)

```bash
=========================== short test summary info ============================
FAILED tests/api/test_api.py::TestApiToken::test_sword_api_can_authenticate - assert 400 == 200
 +  where 400 = <Response [400 Bad Request]>.status_code
================== 1 failed, 79 passed, 65 warnings in 21.12s ==================
```

--- a/pyDataverse/api.py
+++ b/pyDataverse/api.py
@@ -110,7 +110,7 @@ class Api:
         if not isinstance(base_url, str):
             raise ApiUrlError("base_url {0} is not a string.".format(base_url))
 
-        self.base_url = base_url
+        self.base_url = base_url.rstrip("/")
         self.client = None
 
         if not isinstance(api_version, str):

